src/services/target.services.js

```diff
@@ -1,11 +1,11 @@
 import client from '../helpers/client';
 
 const create = (radius, title, topic, lat, lon) => {
-  const user = JSON.parse(localStorage.getItem('user'));
+  const accessToken = localStorage.getItem('accessToken');
   return client.post(
     'targets/',
     JSON.stringify({ radius, title, topic, lat, lon }),
-    { headers: { Authorization: `Bearer ${user.access_token}` } }
+    { headers: { Authorization: `Bearer ${accessToken}` } }
```

Comment:
I think you should remove this and add an interceptor for the requests that automatically adds the token.

By the way, I love how you are managing all of this manually, it'll really help you understand how this stuff works.
Next step might be to sync the token with redux.